src/main.c

```diff
@@ -927,6 +986,17 @@ static void refloat_thd(void *arg) {
                 }
                 d->enable_upside_down = false;
                 d->state.darkride = false;
+
+                // Alert user if cells are out of balance
+                if (bms_get_fault(d->bms_fault, BMSF_CELL_BALANCE)) {
+                    beep_alert(d, 1, false);
+                    d->beep_reason = BEEP_CELL_BALANCE;
+                }
+                // Alert user if bms connection failed.
+                if (bms_get_fault(d->bms_fault, BMSF_CONNECTION)) {
+                    beep_alert(d, 1, true);
+                    d->beep_reason = BEEP_BMS_CONNECTION;
+                }
```

Comment:
I already commented on this on the previous PR, seems to me this will be beeping continuously, so first off, did you test it?

I'll need to think more about the alerting system, and how to get this in now in some sensible way and not have it wait for some sort of alerting revamp.